src/usdb_syncer/utils.py

```diff
@@ -462,3 +463,15 @@ def __exit__(
 
             os.environ.clear()
             os.environ.update(self.modified_env)
+
+
+class JobResult(StrEnum):
```

Comment:
Not really a utility. Would put it in the sql module.

Reply:
In `db.sql.__init__.py`? Or rather `sql.__init__.py`?